src/main/java/frc/robot/Undertaker.java

```diff
@@ -0,0 +1,27 @@
+package frc.robot;
+
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.CANSparkLowLevel.MotorType;
+
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+
+public class Undertaker extends SubsystemBase{
+    
+    private CANSparkMax intakeMotor = new CANSparkMax( Constants.Undertaker.INTAKE_MOTOR_ONE, MotorType.kBrushless);
+    private CANSparkMax intakeMotor2 = new CANSparkMax( Constants.Undertaker.INTAKE_MOTOR_TWO, MotorType.kBrushless);
+    
+    public void undertakerStop(){
+        intakeMotor.set(0);
+        intakeMotor2.set(0);
+    }
+    
+      public void noteRelease(double speed){
```

Comment:
let's rename this `run` and combine the 3 methods. Different commands are what should tell it to run at different speeds to intake, eject, and stop.